components/DescSection.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <section class="section">
+    <div v-if="imageLocation == 1" class="columns is-vcentered">
+      <div class="column">
+        <h1 class="ins-title">{{ title }}</h1>
+        <div v-if="subtitle">
+          <h2 v-for="text in splitComma" :key="text" class="ins-subtitle">
+            {{ text }}
+          </h2>
+        </div>
+        <slot />
+      </div>
+      <div class="column">
+        <img :src="require(`@/assets/${image}`)" :alt="imageDesc" />
+      </div>
+    </div>
+    <div v-else class="columns is-vcentered">
+      <div class="column">
+        <img :src="require(`@/assets/${image}`)" :alt="imageDesc" />
+      </div>
+      <div class="column is-2" />
+      <div class="column">
+        <h1 class="ins-title">{{ title }}</h1>
+        <div v-if="subtitle">
+          <h2 v-for="text in splitComma" :key="text" class="ins-subtitle">
+            {{ text }}
+          </h2>
+        </div>
+        <slot />
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  props: {
+    title: {
+      type: String,
+      required: true
+    },
+    subtitle: {
+      type: String,
+      required: true
+    },
+    image: {
+      type: String,
+      required: true
+    },
+    imageDesc: {
+      type: String,
+      required: true
+    },
+    imageLocation: {
```

Comment:
I propose that the value of imageLocation could be more clear when it says 'left' or 'right'